ApplicationCode/ProjectDataModel/Completions/RimStimPlanFractureTemplate.h

```diff
@@ -125,6 +128,7 @@ class RimStimPlanFractureTemplate : public RimFractureTemplate
     caf::PdmField<QString> m_conductivityResultNameOnFile;
 
     caf::PdmField<double>  m_wellPathDepthAtFracture;
+    caf::PdmField<bool>    m_userDefinedWellPathDepthAtFracture;
```

Comment:
Rename to m_useUserDefinedWellPathDepthAtFracture or m_enableUserDefinedWellPathDepthAtFracture

Reply:
Renamed to m_useUserDefinedWellPathDepthAtFracture